baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/CatchSpecificity.java

```diff
@@ -57,7 +57,7 @@
         name = "CatchSpecificity",
         link = "https://github.com/palantir/gradle-baseline#baseline-error-prone-checks",
         linkType = BugPattern.LinkType.CUSTOM,
-        severity = BugPattern.SeverityLevel.SUGGESTION,
+        severity = BugPattern.SeverityLevel.WARNING,
```

Comment:
I recall there being a reason we had this one set only to suggeation. @carterkozak  do you remember why? or is my memory getting corrupted?

Reply:
Looks like we just wanted to validate that it did the right thing and then planned to increase the severity.
https://github.com/palantir/gradle-baseline/pull/941#discussion_r331304242

Reply:
I think we wanted to roll out the refactors for a while before adding the requirement to avoid friction.